low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -68,6 +68,7 @@ class SSPLBundleGenerate(object):
         self.boot_drvs_dta = f'{sspl_bundle_tmp}/Server_OS_boot_drives_SMART_data_{str(int(time.time()))}'
         self.enclosure_log = f"{sspl_bundle_tmp}/enclosure_logs.zip"
         self.enclosure_wwn = "NA"
+        self.sspl_data_dir = f"/var/{PRODUCT_FAMILY}/sspl/data"
```

Comment:
Sure @cdeshmukh , the same code reflect here as we pushed in cortx-1.0 branch.
can we make those all dynamic config fetch corrections instead of writing static path in https://jts.seagate.com/browse/EOS-17800 or should I do this change here only?

Reply:
take care of it in EOS-17800